clients/include/rocblas_template.yaml

```diff
@@ -13,17 +13,23 @@ Functions:
   rocblas_ddot:  { function: dot, <<: *double_precision  }
   rocblas_sasum: { function: asum, <<: *single_precision }
   rocblas_dasum: { function: asum, <<: *double_precision }
+  rocblas_scasum: { function: asum, <<: *single_precision_complex }
```

Comment:
It looks like I forgot to add all of the added functions here. I'm not sure how this file works exactly, what should be done about zdscal and csscal?